src/core/linter-rules/check-charset.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Linter rule "check-charset".
+ *
+ * Checks whether the document has `<meta charset="utf-8" />` properly.
+ */
+import LinterRule from "../LinterRule";
+import { lang as defaultLang } from "../l10n";
+
+const name = "check-charset";
+const meta = {
+  en: {
+    description: `Document must only contain one \`<meta>\` tag with charset set to 'utf-8'`,
+    howToFix: `Add this line in your document \`<head>\` section - \`<meta charset="utf-8" />\` or set charset to "utf-8" if not set already.`,
```

Comment:
```suggestion
    howToFix: `Add this line in your document \`<head>\` section - \`<meta charset="utf-8">\` or set charset to "utf-8" if not set already.`,
```

Reply:
(Avoid using the deprecated "  \>" xhtml stuff :)

Reply:
(One XML-compliant syntax believer with a sad face)

Reply:
Lol, I never knew! I don't know if we can be friends anymore, @saschanaz 💔 🤣